tests/business/entities/rules/test_time_aggregation_rule.py

```diff
@@ -331,6 +331,78 @@ def test_execute_value_array_aggregate_time_monthly_median():
     )
 
 
+def test_execute_value_array_aggregate_time_monthly_stdev():
+    """Test aggregate input_variable_names of a TimeAggregationRule (QANT10, monthly)"""
```

Comment:
the test is for std, but the description talks about qant10

Reply:
inconsistent as well, the other file talked about quant10

Reply:
oops

Reply:
solved